kafka/src/test/groovy/io/micronaut/configuration/kafka/health/KafkaConsumerMetricsSpec.groovy

```diff
@@ -54,6 +54,16 @@ class KafkaConsumerMetricsSpec extends AbstractEmbeddedServerSpec {
             !result.names.contains("kafka.producer.record-error-rate")
             !result.names.contains("kafka.producer.bytes-consumed-total")
             !result.names.contains("kafka.count")
+
+            def preferredReadReplica = Mono.from(httpClient.exchange("/metrics/kafka.consumer.preferred-read-replica", Map)).block()
+            Map metricBody = preferredReadReplica.body()
+            metricBody.availableTags.size() == 3
+            metricBody.availableTags*.tag == ["partition", "topic", "client-id"]
```

Comment:
@PhilHardwick I'm not seeing `consumer.preferred-read-replica` in the metrics, this seems to give me a 404 in my tests...  What am I missing?

Reply:
See the error over here on the merged-to-master PR: https://github.com/micronaut-projects/micronaut-kafka/pull/513/files#diff-a8766f2290b5d8966c1099f0c7799d85bbc66a3e86557df389966e6900afef24

Reply:
I *think* this fixes it (I found a different metric)

https://github.com/micronaut-projects/micronaut-kafka/pull/513/commits/15a4d791f39a284e59c9b1d976a8cb4339a28e8a